index.bs

```diff
@@ -2814,6 +2814,20 @@ and make sure your choice of algorithm is proven, and up-to-date.
 Not only do they become obsolete or insecure,
 cryptographic protocols and algorithms also evolve quickly.
 
+<h3 id="client-hints">Client Hints must not be used to expose information that is not already 
+exposed to web pages</h3>
+
```

Comment:
```suggestion

When using Client Hints, don't expose new information that the web page does not already know.

```

Reply:
I think this wording is a little funny. How about this?

```suggestion

When using Client Hints, don't expose information that the web page does not already have access to.

```